request-manager/graph/grapher.go

```diff
@@ -361,11 +386,11 @@ func dependenciesSatisfied(inGraph map[string]bool, dependencies []string) bool
 	return true
 }
 
-// Returns a list of node args that aren't present in the job args map.
+// getMissingArgs returns a list of node args that aren't present in the job args map.
 func getMissingArgs(n *spec.Node, jobArgs map[string]bool) ([]string, error) {
```

Comment:
This does more than get the missing, it does all the node arg checks, so s/getMissingArgs/checkNodeArgs/ or similar. And since we really only care about error or not, it can return a single `error` and internally, if there are multiple missing args, it can do like, `return fmt.Errorf("missing args %s", strings.Join(missingArgs, ", "))`